docs/reference/configuration/keywords.rst

```diff
@@ -1425,12 +1424,6 @@ to load, select and modify the analysis modules are documented here.
                 Default is 2.
 
 
-.. _single_node_update:
-.. topic:: SINGLE_NODE_UPDATE
-
```

Comment:
Commit message body should probably describe why `SINGLE_NODE_UPDATE` is removed.